### PR TITLE
CI: Add ingress/egress default deny tests for CNP

### PR DESCRIPTION
**Pull request by manalibhutiyani**
Fixes :#3343
Signed-Off-By: Manali Bhutiyani <manali@covalent.io>

### CODE REVIEW

**Comment by manalibhutiyani** [comment]
test-me-please

**Comment by manalibhutiyani** [comment]
test-missed-k8s

**Comment by ianvernon** [comment]
Retriggering due to the issues with Jenkins we had earlier.

**Comment by ianvernon** [comment]
test-me-please

**Comment by ianvernon** [comment]
test-missed-k8s

**Comment by manalibhutiyani** [comment]
test-me-please

**Comment by manalibhutiyani** [comment]
test-me-please

**Comment by manalibhutiyani** [comment]
test-missed-k8s

File: test/k8sT/Policies.go [hunk 1]
```diff
@@ -406,12 +411,99 @@ var _ = Describe("K8sValidatedPolicyTest", func() {
...
 
 			eps := kubectl.CiliumEndpointPolicyVersion(ciliumPod)
 
 			_, err := kubectl.CiliumPolicyAction(
-				helpers.KubeSystemNamespace, denyEgress, helpers.KubectlApply, 300)
+				helpers.KubeSystemNamespace, knpDenyEgress, helpers.KubectlApply, 300)
+			Expect(err).Should(BeNil(),
+				"L3 deny-egress Policy cannot be applied in %q namespace", helpers.DefaultNamespace)
+
+			err = helpers.WaitUntilEndpointUpdates(ciliumPod, eps, 4, kubectl)
```
Comment (eloycoto) [inline test/k8sT/Policies.go]: This `WaitUntilEndpointUpdates` can be deleted. The CiliumPolicyAction  takes care of that.

File: test/k8sT/Policies.go [hunk 2]
```diff
@@ -406,12 +411,99 @@ var _ = Describe("K8sValidatedPolicyTest", func() {
...
+			By("Installing cnp ingress default-deny")
+
+			eps := kubectl.CiliumEndpointPolicyVersion(ciliumPod)
+
+			_, err := kubectl.CiliumPolicyAction(
+				helpers.KubeSystemNamespace, cnpDenyIngress, helpers.KubectlApply, 300)
+			Expect(err).Should(BeNil(),
+				"L3 deny-ingress Policy cannot be applied in %q namespace", helpers.DefaultNamespace)
+
+			err = helpers.WaitUntilEndpointUpdates(ciliumPod, eps, 4, kubectl)
```
Comment (eloycoto) [inline test/k8sT/Policies.go]: This can be deleted as well.

**Review by eloycoto — CHANGES_REQUESTED** [review]

File: test/k8sT/Policies.go (hunk 1 above)
Comment (manalibhutiyani) [reply test/k8sT/Policies.go]: done

File: test/k8sT/Policies.go (hunk 2 above)
Comment (manalibhutiyani) [reply test/k8sT/Policies.go]: done

**Comment by manalibhutiyani** [comment]
test-me-please

**Comment by manalibhutiyani** [comment]
test-missed-k8s

**Comment by manalibhutiyani** [comment]
Tests passing. Addressed review comments by Eloy. No open review comments. Merging.

**Pull request merged**